parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -301,7 +301,7 @@ if [ "${ENSURE_NO_DUPE_PROMISCUOUS_BRIDGE}" == "true" ]; then
     logs_to_events "AKS.CSE.ensureNoDupOnPromiscuBridge" ensureNoDupOnPromiscuBridge
 fi
 
-if [[ $OS == $UBUNTU_OS_NAME ]]; then
+if [[ $OS == $UBUNTU_OS_NAME || $OS == $MARINER_OS_NAME ]]; then
```

Comment:
nit: separate clauses into their own set of `[]`'s

Reply:
Done